meetings/2017-11-15.md

```diff
@@ -0,0 +1,85 @@
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=eSgqJg7pu_M
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/418
+* **Minutes Google Doc**: https://docs.google.com/document/d/13XLNVP892_NiP_q3sibDG-X2dZABNPBfVZH7tFzT9ng
+
+## Present
+
+* Anna Henningsen @addaleax
+* Сковорода Никита Андреевич @ChALkeR
+* Colin Ihrig @cjihrig
+* Evan Lucas @evanlucas
+* Franziska Hinkelmann @fhinkel
+* Jeremiah Senkpiel @Fishrock123
+* James M Snell @jasnell
+* Matteo Collina @mcollina
+* Myles Borins @MylesBorins
+* Ali Ijaz Sheikh @ofrobots
+* Rod Vagg @rvagg
+* Michaël Zasso @targos
+* Rich Trott @Trott
+
+## Agenda
+
+### Announcements
+
+Myles: NodeFest is coming up in Tokyo soon. A few people from TSC and core project will be there.
+
+James: In about a month, I’ll open the tracking issue for Node.js 10.0.0. Start adding stuff you want in Node.js 10.0.0 to the milestone!
+
+Rod: OpenSSL 1.1.0 port landed in Node.js 9.2.0 yesterday. CI is now testing OpenSSL 1.1.0 on all commits from Node.js 9 onward
+
+### nodejs/node
+
+* Enable explicit `.m.js` intent for ESM [#16170](https://github.com/nodejs/node/pull/16170)
+
+Rich will ask for clarification about exactly what the question/issue is because it doesn’t seem to be the PR itself.
+
+### nodejs/TSC
+
+* doc: initial version of strategic initiatives [#399](https://github.com/nodejs/TSC/pull/399)
+
+No discussion without Michael Dawson.
+
+* Proposal to form a Governance Working Group [#383](https://github.com/nodejs/TSC/issues/383)
+
+No discussion without a scheduled meeting to kick it off.
+
+### nodejs/admin
+
+* meta: fix case of extensions on internal docs and move CoC to admin repo [#25](https://github.com/nodejs/admin/pull/25)
+
+Myles will take next steps. Does not need discussion today.
+
+### nodejs/TSC (again)
+
+* Register Node specific MIME types [#371](https://github.com/nodejs/TSC/issues/371)
+
+Vendor track vs. Standards track
+
+Do we want to ask the board to sponsor it for vendor track? Or pursue it as standards track?
+
+Standards track requires a standards body to write an RFC for the MIME type. It’s a longer road. 
+
+Q: Can the Foundation be used as the contact? 
+
+Jasnell: tl/dr; version: Mime types can be registered Standards track or Vendor track as described here: https://tools.ietf.org/html/rfc4288#section-3.1. Vendor track is for commercial products, Standards track is for stuff that is of “general”interest. The IESG/IETF may be viewing this in the latter case -- in general, they prefer things *not* to be vendor track as much as possible. The concern is that Standards track takes longer and requires an RFC to be published, which any one of us could write up. It doesn’t require Board action to push through, so we should get Bradley’s reasoning for why he’d like to see that. Also, fwiw, I’ve written many RFC’s to this point and I’m quite familiar with the process so can help push this forward.
```

Comment:
Missing space between `"general"` and `interest`.

One lowercase `vendor track`.